### PR TITLE
feat(ZENKO-1313): add Cosmos chart

### PR DESCRIPTION
**Pull request by giacope**
**What does this PR do, and why do we need it?**

This adds the Cosmos 🌌chart to Zenko.

**Special notes for your reviewers**:

See https://github.com/scality/Zenko-RFCs/pull/5

### CODE REVIEW

**Comment by giacomoguiulfo** [comment]
@bert-e wait

**Comment by ghost** [comment]
There were the following issues with this Pull Request

* Commit: c28048500d1dc5abf4fc1c6aad944a652aa4f06a
  - ✖ message may not be empty
  - ✖ type may not be empty


You may need to [change the commit messages][ref] to comply with the repository contributing guidelines.

--------

🤖 This comment was generated by [**commitlint[bot]**][repo]. Please report issues [here][issues].

Happy coding!

[ref]: https://help.github.com/articles/changing-a-commit-message/
[repo]: https://github.com/z0al/commitlint-bot
[issues]: https://github.com/z0al/commitlint-bot/issues

File: kubernetes/cosmos/README.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+4. Configure the `rcd.remote` values in the `values.yaml` file. For example:
+
+```yaml
+rcd:
+  remote:
+    accessKey: my-access-key
+    secretKey: my-secret-key
+    endpoint: http://cloudserver.local
+    region: us-east-2
```
Comment (ssalaues) [inline kubernetes/cosmos/README.md]: We will need to expose the ingestion bucket parameter to comply with orbit and other MD ingestion features.

**Comment by vrancurel** [comment]
can we have a simple end-to-end test in this PR ?

**Comment by giacope** [comment]
@vrancurel Unfortunately, we can't test cosmos functionality with the current CI setup. However, I added a few steps in eve to verify that the cosmos chart lints and templates properly.

File: kubernetes/cosmos/values.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+pfsd:
+  name: pfsd
+
+  replicaCount: 1
+
+  image:
+    repository: gguiulfo/cloudserver
```
Comment (ssalaues) [inline kubernetes/cosmos/values.yaml]: can we use an official image now?

File: kubernetes/cosmos/values.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  resources: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+
+rclone:
+  name: rclone
+
+  image:
+    repository: gguiulfo/rclone
```
Comment (ssalaues) [inline kubernetes/cosmos/values.yaml]: same with this

**Review by tmacro — APPROVED** [review]

File: kubernetes/cosmos/values.yaml (hunk 3 above)
Comment (giacope) [reply kubernetes/cosmos/values.yaml]: Let's wait for https://github.com/scality/Zenko/pull/464 to use that image instead.

File: kubernetes/cosmos/values.yaml (hunk 2 above)
Comment (giacope) [reply kubernetes/cosmos/values.yaml]: updated it!

File: kubernetes/cosmos/values.yaml (hunk 3 above)
Comment (giacope) [reply kubernetes/cosmos/values.yaml]: Done!

**Comment by giacope** [comment]
@ssalaues Updated the images

**Review by ssalaues — APPROVED** [review]

**Comment by giacope** [comment]
@bert-e approve

**Comment by giacope** [comment]
@bert-e help

**Comment by giacope** [comment]
@bert-e help

**Comment by giacope** [comment]
@bert-e help

**Comment by ssalaues** [comment]
@bert-e reset

**Comment by ssalaues** [comment]
@bert-e reset

**Comment by ssalaues** [comment]
The queue was failing because Eve thought it wasn't at the branch tip. I deleted the queue branch and forced a rebuild. Should merge now.

**Pull request merged**